package.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>pfs</name>
+  <version>0.0.0</version>
```

Comment:
`0.10.0`?

Reply:
I've updated the version to 0.11.0 because that is the latest tag. Is that OK?